src/participants/create_project.py

```diff
@@ -148,55 +153,80 @@ def handle_wi(self, wid):
             create = True
 
         project_list = self.obs.getProjectList()
-        #if project in project_list:
+        # if project in project_list:
             # project already exists, don't do anything
         #    return
 
         prj_parts = project.split(":")
         if prj_parts[0] == "home" and len(prj_parts) > 1:
             maintainers.append(project.split(":")[1])
-            if project not in project_list: create = True
-            #TODO: construct repos and build paths for a devel build
+            if project not in project_list:
+                create = True
+            # TODO: construct repos and build paths for a devel build
 
         if len(prj_parts) >= 3 and prj_parts[-3] == "feature":
-            link = ":".join(prj_parts[0:-3])
+            linked_project = ":".join(prj_parts[0:-3])
             fea = "%s#%s" % (prj_parts[-2], prj_parts[-1])
             # Go through each bugzilla we support
             for (bugzillaname, bugzilla) in self.bzs.iteritems():
                 for match in bugzilla['compiled_re'].finditer(fea):
                     try:
-                        summary = bugzilla['interface'].bug_get(match.group('key'), 0)['summary']
-                        desc = bugzilla['interface'].comment_get(match.group('key'), 0)['text']
+                        summary = bugzilla['interface'].bug_get(
+                            match.group('key'), 0)['summary']
```

Comment:
There are several cases of breaking a line leaving an opening paren as the last character, and then having the closing paren on the following line.  In lisp that's readable but in Python I don't like it, and this is one of the more egregious examples for why:  "bug_get(".  Eye scans for closing paren, what, it close with "]"? Is this a SyntaxError?  Ah no, it's ")['summary']".

I much prefer the style where the closing paren is balanced with the opening one, like:

     summary = bugzilla['interface'].bug_get(
          match.group('key'), 0
     )['summary']